### ISSUE TITLE
List of invalid fields

### ISSUE THREAD

**Issue by Razi91** [kazupon/vue-validator#51]
For debug purposes and easier listing invalid filled fields: list of invalid fields.

For example:

validation method:

```
validName: {
    message: function(el){
        return "between 5 and 10 characters";
    },
    check: function(v){
        return v.length >= 5 && v.length <= 10;
    }
}
```

HTML:
`<input v-model="form.name"
    name="companyName"
    v-validate="validName"/>`

`````` <ul>
<li v-for="error in validation.fields"><span v-text="error.fieldName"></span>: <span v-text="error.message"></span></li>
</ul>``` 

where `error.fieldName` is input's name and `error.message` contains what `message` method returns (element given as parameter
``````

**Comment by kazupon** [comment]
Hi @Razi91 !!

Thank you for your reporting.
Can you provide the issue code on the jsfiddle ?

**Comment by Razi91** [comment]
It's more like feature proposition.

**Comment by kazupon** [comment]
ah, :sweat: 
I see.

**Comment by kazupon** [comment]
I'll try to consider this feature at #41

**Comment by kazupon** [comment]
Published v2.0.0-alpha.9
https://github.com/vuejs/vue-validator/releases/tag/v2.0.0-alpha.9
